Aplib.Core/Tactics/FirstOfTactic.cs

```diff
@@ -8,22 +8,31 @@ namespace Aplib.Core.Tactics
     /// </summary>
     public class FirstOfTactic : Tactic
```

Comment:
Is FirstOfTactic not a more specific AnyOfTactic. Would it be better to subclass from that? Then you don't have to define Subtactics twice.